Documentation/Updating.md

```diff
@@ -1,13 +1,13 @@
-# Updating to the General Availability (GA) Release
+# Updating to the 2.0.0 Release
 
-Between the RC2 and GA releases of the Microsoft Mixed Reality Toolkit, changes were made that may impact existing projects. This document describes those changes and how to update projects to the GA release.
+Between the RC2 and 2.0.0 releases of the Microsoft Mixed Reality Toolkit, changes were made that may impact existing projects. This document describes those changes and how to update projects to the 2.0.0 release.
 
 - [API changes](#api-changes)
 - [Assembly name changes](#assembly-name-changes)
 
 ## API changes
 
-Since the release of RC2, there have been a number of API changes including some that may break existing projects. The following sections describe the changes that have occurred between the RC2 and GA releases.
+Since the release of RC2, there have been a number of API changes including some that may break existing projects. The following sections describe the changes that have occurred between the RC2 and 2.0.0 releases.
 
 - [MixedRealityToolkit](#mixedrealitytoolkit)
 - [Event System](#event-system)
```

Comment:
The TOC list here also doesn't match up with the actual headers shown on the right hand side

Reply:
@Troy-Ferrell, not sure what you mean here, can you give an example? thanks!

Reply:
There is that list of internal hyperlinks that describes the sections but it doesn't exactly match up with the list of sections outlined on the right: 

![image](https://user-images.githubusercontent.com/25975362/62982755-1db26580-bde2-11e9-9944-a789a884750c.png)

Reply:
i see what you mean now... thanks!

Reply:
added missing headings